src/sysinfo/nvapi_adapter.cpp

```diff
@@ -147,7 +157,51 @@ namespace dxvk {
         return 0;
     }
 
-    bool NvapiAdapter::isVkDeviceExtensionSupported(const std::string name) { // NOLINT(performance-unnecessary-value-param)
+    bool NvapiAdapter::GetLUID(LUID *luid) const {
+        if (m_deviceIdProperties.deviceLUIDValid) {
```

Comment:
That was the part I was the most curious about. I did the same in my implementation, but windows games didn't quite like it. When I tried to mock the windows behaviour of getting the LUID by hardcoding the same value, it worked better. Hence the question is this supposed to work (I am not that much of a graphics developer)? When I was returning the `deviceLUID` received from vulkan the games in wine never liked the value, but when I hardcoded it to like `0x3f7` or other constants I extracted from windows, it worked better. I know it feels like the output should be the same, but I don't know what nvidia driver for windows uses to obtain the LUID. I might need to recheck, but I thought that it simply used directx API to get it, like [that one](https://docs.microsoft.com/en-us/windows/win32/api/d3d12/nf-d3d12-id3d12device-getadapterluid).  Am I wrong here? Cuz I hope I am and this vulkan's LUID should work exactly the same as in directx/dxgi.

Reply:
I think I was just doing it wrong, `0x3f7` is also returned from here and passes the same checks later.

Reply:
> I think I was just doing it wrong, `0x3f7` is also returned from here and passes the same checks later.

Thanks for the confirmation that the implementation works. That said, please keep the discussion here focused to the PR itself ;)